oak/server/oak_main.cc

```diff
@@ -48,13 +49,18 @@ int main(int argc, char* argv[]) {
       absl::make_unique<oak::OakManager>(absl::GetFlag(FLAGS_enclave_path));
 
   // Initialize and run gRPC server.
-  LOG(INFO) << "Starting gRPC server";
+  int selected_port = absl::GetFlag(FLAGS_grpc_port);
+  std::string addr = "[::]:" + std::to_string(selected_port);
+  LOG(INFO) << "Starting gRPC server on " << addr;
   grpc::ServerBuilder builder;
-  int selected_port;
-  builder.AddListeningPort("[::]:8888", grpc::InsecureServerCredentials(), &selected_port);
+  builder.AddListeningPort(addr, grpc::InsecureServerCredentials(), &selected_port);
```

Comment:
I think this is going to write the selected port back into `selected_port`. I think we should either pass `nullptr` if it's allowed, or actually use (at least log) the output value.

Reply:
not actually reverted?

Reply:
Weird &ndash; GitHub was including the [commit](https://github.com/project-oak/oak/pull/104/commits/b0ef4af5693bd09a7ecd698ae8e17dfdf2cf0277) that changes the log but wasn't including it in the overall diff.  But now it is!

Reply:
Still not seeing the diff reverted, not sure what's going on.

Reply:
Do you still not see the original `LOG` line now I've rebased and de-conflicted?  I see:

![image](https://user-images.githubusercontent.com/457208/59512643-0e31a180-8eb1-11e9-94a0-0063b86b191b.png)

Reply:
Ah yes sorry, my bad, it's there.

Reply:
I think it's GitHub's bad &ndash; I also had the old diff showing up (despite the new commit), and then it mysteriously caught up (without anything changing in the repo in the meanwhile).